ext/spl/spl_array.c

```diff
@@ -1918,7 +1918,7 @@ ZEND_BEGIN_ARG_INFO(arginfo_array_seek, 0)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO(arginfo_array_exchangeArray, 0)
-	ZEND_ARG_INFO(0, array)
+	ZEND_ARG_INFO(0, input)
```

Comment:
Should update https://github.com/php/php-src/blob/0fe767d559cb52f38433e0e1c15c9f299058cca8/ext/spl/spl_array.c#L1293.